internal/bundler/bundler.go

```diff
@@ -94,6 +94,14 @@ func (b *Bundler) Bundle() (string, error) {
 		}
 	}
 
+	// List contents of archive in verbose mode for easier debugging
+	// when we do not have access to the bundle itself
+	var content []string
+	for _, h := range archiveWriter.Headers() {
+		content = append(content, fmt.Sprintf("%s %12d %s", h.FileInfo().Mode().String(), h.Size, h.Name))
+	}
+	log.Debugf("Archive content:\n%s", strings.Join(content, "\n"))
```

Comment:
We can use the builtin `tabwriter` package here to ensure that the width of the columns fits the string lengths:
```suggestion
	w := tabwriter.NewWriter(os.Stderr, 0, 0, 1, ' ', tabwriter.AlignRight)
	for _, h := range archiveWriter.Headers() {
		_, err := fmt.Fprintf(w, "%s\t%d\t %s\n", h.FileInfo().Mode().String(), h.Size, h.Name)
		if err != nil {
			return "", errors.WithStack(err)
		}
	}
	log.Debugf("Content of bundle %s:", bundle.Name())
	err = w.Flush()
	if err != nil {
		return "", errors.WithStack(err)
	}
```